frontend/src/metabase/admin/tasks/containers/Help.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from "react";
+import { t } from "ttag";
+import _ from "underscore";
+
+import { Box } from "grid-styled";
+import AdminHeader from "metabase/components/AdminHeader";
+import CopyButton from "metabase/components/CopyButton";
+import ExternalLink from "metabase/components/ExternalLink";
+
+import { UtilApi } from "metabase/services";
+
+function navigatorInfo() {
+  return _.pick(navigator, "language", "platform", "userAgent", "vendor");
+}
+
+const template = `**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Logs**
+Please include javascript console and server logs around the time this bug occurred. For information about how to get these, consult our [bug troubleshooting guide](https://metabase.com/docs/latest/troubleshooting-guide/bugs.html)
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Severity**
+How severe an issue is this bug to you? Is this annoying, blocking some users, blocking an upgrade or blocking your usage of Metabase entirely?
+Note: the more honest and specific you are here the more we will take you seriously.
+
+**Additional context**
+Add any other context about the problem here.
+
+**Metabase Diagnostic Info**
+`;
+
+function githubIssueLink(bugReportDetails) {
+  return (
+    "https://github.com/metabase/metabase/issues/new?title=&labels=Type:Bug&body=" +
+    encodeURI(template) +
```

Comment:
Let's change this to 

```js
encodeURIComponent(template + "\n```json\n" + bugReportDetails + "\n```")
```

so that the JSON object is formatted nicely in the issue.

Also using `encodeURIComponent` will escape semicolons (and probably other characters) correctly, so you won't need to strip them.